metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
I'm inclined to agree that we should absolutely track this down, but since you didn't introduce those dependencies in this PR, it doesn't seem right to hold up this PR until we do. I'd recommend opening an additional Jira to address the libraries of concern and treat this as a separate issue.